samples/authSample.html

```diff
@@ -13,83 +13,104 @@
   License for the specific language governing permissions and limitations under
   the License.
 
-  To run this sample, replace YOUR API KEY with your application's API key.
-  It can be found at https://code.google.com/apis/console/?api=plus under API Access.
-  Activate the Google+ service at https://code.google.com/apis/console/ under Services
+  To run this sample, set apiKey to your application's API key and clientId to
+  your application's OAuth 2.0 client ID. They can be generated at:
+    https://console.developers.google.com/apis/credentials?project=_
+  Then, add a JavaScript origin to the client that corresponds to the domain
+  where you will be running the script. Finally, activate the People API at:
+    https://console.developers.google.com/apis/library?project=_
 -->
 <!DOCTYPE html>
 <html>
   <head>
     <meta charset='utf-8' />
   </head>
   <body>
-    <!--Add a button for the user to click to initiate auth sequence -->
-    <button id="authorize-button" style="visibility: hidden">Authorize</button>
+    <p>Say hello using the People API.</p>
+
+    <!--Add buttons to initiate auth sequence and sign out-->
+    <button id="authorize-button">Authorize</button>
```

Comment:
The authorize button should be initially invisible or non-displayed so that disallowing JavaScript execution or problems loading or running the sign-in libraries do not result in a nonfunctional-yet-visible UI element.